deployment.md

```diff
@@ -0,0 +1,164 @@
+# Deployment & CLI
+
+OpenMined PyGrid CLI provide the necessary support for Infrastructure and Deployment management of various PyGrid apps including (Domains, Networks, and Workers) to the main cloud providers ([AWS](https://aws.amazon.com/), [GCP](https://cloud.google.com/), [Azure](https://azure.microsoft.com/)) through the usage of [Terraform](https://www.terraform.io/) and [TerraScript](https://github.com/mjuenema/python-terrascript) integrated within the CLI.
+
+## Installing Terraform
+
+Check Instructions here:
+- https://www.terraform.io/downloads.html
+- https://learn.hashicorp.com/tutorials/terraform/install-cli
+
+
+## Installing the CLI
+
+To get started, install the CLI first through thess commands:
+
+Clone PyGrid Repo:
+
+```shell
+git clone https://github.com/OpenMined/PyGrid.git
+```
+
+Then install the python package through pip
+
+```shell
+pip install -e .
+```
+
+## CLI Instructions
+
+Using the PyGrid CLI is very simple through this command `pygrid deploy` then following the instruction to successfully deploying a specific pygrid app, described in the commands below
+
+### List of CLI Commands
+
+#### Deploy a Domain to AWS
+
+```shell
+pygrid deploy --provider aws --app domain
+```
+
+#### Deploy a Network to Azure
+
+```shell
+pygrid deploy --provider azure --app network
+```
+
+CLI Instructions Example
+
+```shell
+$ pygrid deploy                                                                                                                                           [7h 19m]
```

Comment:
If I install `pygrid` following your instructions (`pip install -e .`), `pygrid deploy` raises the following error:

`ModuleNotFoundError: No module named 'requests'`

Reply:
Ahaa `requests` library was used before in the old CLI, but not anymore

Reply:
Thanks for pointing this out! :)